assets/js/src/form_editor/store/actions.jsx

```diff
@@ -28,3 +28,20 @@ export function saveFormDone(result) {
     result,
   };
 }
+
+export function addNotice(content, status, isDismissible = false, id = null) {
```

Comment:
I am not sure why is the `id` argument optional.

Reply:
It is just for convenience if you don't pass any it will be generated, but this is not needed for now. This notification system will evolve as we add new notifications so I'll make it non-optional for now.